src/app/ASKS/ask/index.js

```diff
@@ -722,6 +722,23 @@ const Ask = ({
           clientId={clientId}
         />
       )}
+      {component === 'time' && (
```

Comment:
Why does this PR have changes regarding to the time component? Can you please remove all of it.

Reply:
Fixed in latest commit